### PR TITLE
Update yue

### PR DESCRIPTION
**Pull request by Frityet**

### CODE REVIEW

**Comment by carsakiller** [comment]
@Frityet a check has failed

**Comment by Frityet** [comment]
@carsakiller should fix it, its now one file instead of 100+

File: addons/tex-luametatex/module
Comment (carsakiller) [inline addons/tex-luametatex/module:1]: Please re-pull the latest changes that include the changes to this module.

File: addons/tex-luatex/module
Comment (carsakiller) [inline addons/tex-luatex/module:1]: This module is also out of date in this PR.

**Review by carsakiller — CHANGES_REQUESTED** [review]
Your branch is out of date and does not include some recent changes to LLS-Addons. Please update your branch and rebase your changes.

File: addons/tex-luatex/module
Comment (Frityet) [reply addons/tex-luatex/module:1]: whoops, thanks

**Pull request closed without merging**